src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -162,5 +162,11 @@ public int MaxModelValidationErrors
         /// Gets or sets the default value for the Permanent property of <see cref="RequireHttpsAttribute"/>.
         /// </summary>
         public bool RequireHttpsPermanent { get; set; }
+
+        /// <summary>
+        /// Gets or sets the flag to allow binding undefined values to Enum types.
```

Comment:
Suggest `Gets or sets an indication whether the model binding system will bind undefined values to enumeration types.` Note the formal "enumeration types" because that's suitable for translation.